lib/redux/middleware/sentry_api_middleware.dart

```diff
@@ -150,16 +150,18 @@ class SentryApiMiddleware extends MiddlewareClass<AppState> {
               organizationSlug: action.organizationSlug,
               projectId: action.projectId,
               fields: [SessionGroup.sumSessionKey, SessionGroup.countUniqueUsersKey],
-              groupBy: SessionGroupBy.sessionStatusKey
```

Comment:
Wouldn't we be loosing out on showing issues that occured in < 1h wen we do it that way?

Reply:
That is true, but showing incomplete hour data is always confusing. One can say if you look at it at e.g. 9:10 -> Cool I don't have that many errors anymore but the reason for that is just because the hour isn't over yet.